plugins/woocommerce-blocks/assets/js/blocks/product-gallery/utils.tsx

```diff
@@ -136,10 +136,10 @@ export const moveInnerBlocksToPosition = (
 ): void => {
 	const { getBlock, getBlockRootClientId, getBlockIndex } =
 		select( 'core/block-editor' );
+	const { moveBlockToPosition } = dispatch( 'core/block-editor' );
```

Comment:
Nitpick but what's the reason for this change? I think it's a good approach to initiate variable only when it's going to be used. If anywhere, I'd move to this `if` statement as it's only used in this particular case:

https://github.com/woocommerce/woocommerce/blob/9f57a9a7b7d93a2e0ddb78e117c52fc9050f4072/plugins/woocommerce-blocks/assets/js/blocks/product-gallery/utils.tsx#L212